robotidy/transformers/SplitTooLongLine.py

```diff
@@ -0,0 +1,112 @@
+from itertools import tee
+
+from robot.api.parsing import (
+    ModelTransformer,
+    Token
+)
+
+
+def eol():
+    return Token(Token.EOL)
+
+
+class SplitTooLongLine(ModelTransformer):
+    def __init__(self, line_length: int = 120):
+        self.line_length = line_length
+
+    def visit_KeywordCall(self, node):  # noqa
+        if all(token.end_col_offset <= self.line_length for token in node.tokens[::-1]):
+            return node
+        if not node.get_tokens(Token.ARGUMENT):  # return if there are no arguments - nothing to split
+            return node
+        return self.split_keyword_call(node)
+
+    def split_keyword_call(self, node):
+        is_single_line = node.lineno == node.end_lineno
+
+        indent = node.tokens[0]
+        assignment = list(self.insert_seperator(node.get_tokens(Token.ASSIGN)))
+        keyword = node.get_token(Token.KEYWORD)
+
+        if is_single_line:
+            comment = node.get_tokens(Token.COMMENT)
+            comment_line = [indent] + comment + [eol()] if comment else []
+
+            head = comment_line + [indent] + assignment + [keyword]
+
+            tail = node.tokens[node.tokens.index(keyword) + 1:]
+            tail = list(filter(lambda t: t.type != Token.COMMENT, tail))
```

Comment:
how about just:
```
tail = [token for node.tokens[node.tokens.index(keyword) + 1:] if token.type != Token.COMMENT]
```

Reply:
Yup, this line was an afterthought fix, didn't managed to refactor it before sending PR. Will do.